src/general_manager/rule/handler.py

```
@@ -3,18 +3,20 @@
 from __future__ import annotations
 import ast
 from typing import Dict, Optional, TYPE_CHECKING, cast
+from abc import ABC, abstractmethod
 
 if TYPE_CHECKING:
     # Forward-Reference auf Rule mit beliebigem Generic-Parameter
     from general_manager.rule.rule import Rule
     from general_manager.manager import GeneralManager
```

Ruff (0.8.2)

`general_manager.manager.GeneralManager` imported but unused

Remove unused import: `general_manager.manager.GeneralManager`

(F401)